collector/compile-benchmarks/README.md

```diff
@@ -25,8 +25,8 @@ They mostly consist of real-world crates.
   part of the Rust ecosystem.
 - **cargo-0.87.1**: The Rust package manager. A large program, and an important
   part of the Rust ecosystem.
-- **clap-3.1.6**: A command line argument parser library. A crate used by many
-  Rust programs.
+- **clap-3.1.6**: A command line argument parser library. A crate used by many Rust programs.
+- **clap-4.5.36**: A command line argument parser library. A crate used by many Rust programs.
```

Comment:
The description should be updated to explain that it's just the proc macro part.